code_samples/api/product_catalog/src/Command/ProductPriceCommand.php

```diff
@@ -18,6 +24,8 @@ final class ProductPriceCommand extends Command
 {
     private ProductPriceServiceInterface $productPriceService;
 
+    private PriceResolverInterface $priceResolver;
```

Comment:
We are missing constructor parameter and its assignment:
```
public function __construct(
    CurrencyServiceInterface $currencyService,
    ProductServiceInterface $productService,
    ProductPriceServiceInterface $productPriceService,
    PriceResolverInterface $priceResolver,
    UserService $userService,
    PermissionResolver $permissionResolver
) {
    $this->currencyService = $currencyService;
    $this->productPriceService = $productPriceService;
    $this->priceResolver = $priceResolver.
    $this->productService = $productService;
    $this->userService = $userService;
    $this->permissionResolver = $permissionResolver;
 
    parent::__construct('doc:price');
}

Reply:
Please see the updated section below